policy-engine/src/graph.rs

```diff
@@ -60,17 +61,30 @@ async fn _index(
     let path = req.uri().path();
     GRAPH_INCOMING_REQS.with_label_values(&[path]).inc();
 
+    let mut generic_header: Vec<actix_web::http::HeaderValue> =
+        vec![header::HeaderValue::from_static(CONTENT_TYPE)];
+
+    let mut version_headers: Vec<actix_web::http::HeaderValue> = commons::CINCINNATI_VERSION
+        .keys()
+        .map(|val| header::HeaderValue::from_static(val))
+        .collect();
+
+    version_headers.append(&mut generic_header);
+
     // Check that the client can accept JSON media type.
-    commons::validate_content_type(req.headers(), CONTENT_TYPE)?;
+    let cincinnati_version: String =
+        commons::validate_content_type(req.headers(), version_headers)?;
 
     // Check for required client parameters.
     let mandatory_params = &app_data.mandatory_params;
     commons::ensure_query_params(mandatory_params, req.query_string())?;
 
-    let plugin_params = Query::<HashMap<String, String>>::from_query(req.query_string())
+    let mut plugin_params = Query::<HashMap<String, String>>::from_query(req.query_string())
         .map(|query| query.into_inner())
         .map_err(|e| commons::GraphError::InvalidParams(e.to_string()))?;
 
+    plugin_params.insert(String::from("version"), cincinnati_version);
```

Comment:
you're injecting the media type here, not the version integer, right?  Probably call this `content_type` or some such (or convert it to the version integer/string, without the rest of the `application/vnd.redhat.cincinnati...` context).